community-membership.md

```diff
@@ -1,63 +1,99 @@
+| Committer | Review contributions from other members | Finish at least one major contributions, History of reviews| [COMITTER](https://github.com/milvus-io/milvus/blob/master/OWNERS) |
+| Maintainer | Contributions acceptance approval| Highly experienced active reviewer and contributor| [MAINTAINERS](https://github.com/milvus-io/milvus/blob/master/OWNERS_ALIASES)|
+
+## Roles and Responsibilities
+
+### Contributors
+Contributors are individuals who actively make contribution and are willing to participate in the code review of new contributions. They can have
+issues and PRs assigned to them, participate in discussion through GitHub and slack, and pre-submit tests are automatically run for their PRs. 
+
+**Defined by:** Github all contributor list
+
+#### Requirements
+- Have made at least one contribution to the project or community.  Contribution may include, but is not limited to:
+    - Authoring or reviewing PRs on GitHub. At least one PR must be **merged**.
+    - Filing or commenting on issues on GitHub
+    - Contributing to community discussions (e.g. meetings, Slack, github discussion)
+    - Improve docs
+  
+- Subscribed to the community [slack](milvusio.slack.com)
+- Have read the [contributor guide](https://github.com/milvus-io/community/blob/master/CONTRIBUTING.md)
+- Actively contributing to 1 or more milvus related repos.
+
+#### Responsibilities and privileges
+
+- Expected to be responsive to issues and PRs assigned to them
+- Expected to follow community rules and values
+- They can be assigned to issues and PRs, and people can ask contributors for reviews with a `/assign @username`.
+- Tests can be run against their PRs automatically. 
+- Ask for mentorship from Committers/Maintainers
+- Join the community meetings and give their advice
+
+### Committer
+Committers are able to review code for quality and correctness on some part of the project. They are knowledgeable about both the codebase and software
+engineering principles and they should pratice the community core values and guidelines as an example.
+
+**Defined by:** [COMITTER](https://github.com/milvus-io/milvus/blob/master/OWNERS)
+
+#### Requirements
+
+- contributor for at least 3 months
+- Finish at least one major contribution to the community
+- Reviewers for at least 5 PRs to the codebase
+- Reviewed or merged at least 5 substantial PRs to the codebase
+- Knowledgeable about the codebase/doc
+- Nominated by at least 1 Maintainer, seconded by at least 3 other committers/maintainers.
+
+#### Responsibilities and privileges
+- Expected to be responsive to review pull requests
+- Expected to assign issues and prs to related expertise
+- Expected to be responsive to mentions of others on github and slack
+- Expected to join the community meetings and activities in time
+- Committer status may be a precondition to accepting large code contributions
+- Mentor and guide new contributors
+- Committers can do `/lgtm` on open PRs.
```

Comment:
For now, `/lgtm` is controlled by prow- https://prow.zilliz.cc/command-help#lgtm , all Collaborators on the repository can do.

`/approve` -https://prow.zilliz.cc/command-help#approve-Users listed as 'approvers' in appropriate OWNERS files
Does this mean we need do something to change current situation? @LoveEachDay  please help review if I have any misunderstanding.

Reply:
committers can only do `/lgtm`, mainteners can do `/approve`. 
for now, we don't need any changes.